src/main/java/rx/internal/util/BackpressureDrainManager.java

```diff
@@ -174,7 +171,7 @@ public final void drain() {
             emitting = true;
             term = terminated;
         }
-        n = requestedCount;
+        n = requestedCount.get();
```

Comment:
If not inlined, requestedCount reference should be read into a local variable outside the loop.